arviz/plots/distplot.py

```diff
@@ -80,24 +80,37 @@ def plot_dist(
         Figure size. If None it will be defined automatically.
     textsize: float
         Text size scaling factor for labels, titles and lines. If None it will be autoscaled based
-        on figsize. Not implemented for bokeh backend.
+        on ``figsize``. Not implemented for bokeh backend.
     plot_kwargs : dict
         Keywords passed to the pdf line of a 1D KDE.
+        Passed to :func:`arviz.plot_kde` as ``plot_kwargs``.
     fill_kwargs : dict
         Keywords passed to the fill under the line (use fill_kwargs={'alpha': 0} to disable fill).
-        Ignored for 2D KDE
+        Ignored for 2D KDE. Passed to :func:`arviz.plot_kde` as ``fill_kwargs``.
     rug_kwargs : dict
         Keywords passed to the rug plot. Ignored if rug=False or for 2D KDE
-        Use `space` keyword (float) to control the position of the rugplot. The larger this number
-        the lower the rugplot.
+        Use ``space`` keyword (float) to control the position of the rugplot.
+        The larger this number the lower the rugplot. Passed to
+        :func:`arviz.plot_kde` as ``rug_kwargs``.
     contour_kwargs : dict
         Keywords passed to the contourplot. Ignored for 1D KDE.
     contourf_kwargs : dict
-        Keywords passed to ax.contourf. Ignored for 1D KDE.
+        Keywords passed to :meth:`matplotlib.axes.Axes.contourf`. Ignored for 1D KDE.
     pcolormesh_kwargs : dict
-        Keywords passed to ax.pcolormesh. Ignored for 1D KDE.
+        Keywords passed to :meth:`matplotlib.axes.Axes.pcolormesh`. Ignored for 1D KDE.
     hist_kwargs : dict
-        Keywords passed to the histogram.
+        Keyword arguments used to customize the histogram. Ignored when plotting a KDE.
+        They are passed to :meth:`matplotlib.axes.Axes.hist` if using matplotlib,
+        or to :meth:`bokeh.plotting.Figure.quad` if using bokeh. In bokeh case,
+        the following extra keywords are also supported:
+
+        * ``color``: replaces the ``fill_color`` and ``line_color`` of the ``quad`` method
+        * ``bins``: taken from ``hist_kwargs`` and passed to :func:`numpy.histogram` instead
+        * ``density``: normalize histogram to represent a probability density function,
+        Defaults to ``True``
```

Comment:
The indentation is not right and renders the default as a new paragraph outside the list. It should be at the same level as the start of the text in the bullet point, not at the same level as the bullet point icon

Reply:
I tried adding indentation but it was failing the checks and messing up the UI.

Reply:
how many spaces?

Reply:
Done, please check.